### ISSUE TITLE
Matlab 2017b: Name conflict warning for deprecated vecnorm.m

### ISSUE THREAD

**Issue by oqilipo** [mattools/matGeom#40]
I've just installed 2017b and receive the following warning:

```
Warning: Function vecnorm has the same name as a MATLAB builtin. 
We suggest you rename the function to avoid a potential name conflict. 
``` 

May it be possible to remove the function from the toolbox?
``` 
matGeom\matGeom\deprecated\geom2d
``` 

Kind regards

**Comment by dlegland** [comment]
Hi,
yes, it I kept it in a deprecated directory to avoid ascent compatibility questions, but removing it should be better now.

**Comment by dlegland** [comment]
fixed in commit 91c0196f4217e97b02db740d4415826f9da9abc2
